shells/android/java/arcs/storage/service/StorageService.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.storage.service
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+import arcs.storage.parcelables.ParcelableModelUpdate
+import arcs.storage.parcelables.ParcelableOperations
+import arcs.storage.parcelables.ParcelableSyncRequest
+
+class StorageService : Service() {
```

Comment:
I think either this class, or `IStorageService`, should have good documentation. I think probably `IStorageService`, because it is what clients will actually be using.

Reply:
100% agree, everything will be well documented when I'm working on the PR with actual functionality... This PR only includes these stubs and tentative APIs as a way to ensure things can build.